### PR TITLE
boards: riscv: ite: enable the CONFIG_OUTPUT_PRINT_MEMORY_USAGE

### LINKED ISSUE #50165: boards: riscv: ite: No flash and RAM stats are shown whenever building ITE board
**Describe the bug**
No flash and RAM stats are shown whenever building ITE board

it8xxx2_evb

**To Reproduce**
1. cd samples/helloworld
2. rm -rf build && west build -c -b it8xxx2_evb
3.  No flash and RAM stats are shown.

**Expected behavior**
Flash and RAM stats should be shown to support footprint assessments

**Impact**
This prevent RAM and code footprint assessments, planning and optimizations.

**Logs and console output**
-- west build: building application
[1/150] Preparing syscall dependency handling
[3/150] Generating include/generated/version.h
-- Zephyr version: 3.2.0-rc1 (/home/jamezaar/ecwork/zephyr_albertofloyd), build: zephyr-v3.1.0-5147-g7c50ab1f0364
[140/150] Linking C executable zephyr/zephyr_pre0.elf
[144/150] Linking C executable zephyr/zephyr_pre1.elf
[150/150] Linking C executable zephyr/zephyr.elf

**Environment (please complete the following information):**
Linux

### PR DESCRIPTION
**Pull request by bjchentc**
Removed the CONFIG_OUTPUT_PRINT_MEMORY_USAGE in it8xxx2_evb_defconfig

fixes #50165

Signed-off-by: BJ Chen <bj.chen@ite.com.tw>

### CODE REVIEW

**Review by stephanosio — DISMISSED** [review]

**Review by carlocaione — DISMISSED** [review]

**Review by keith-zephyr — DISMISSED** [review]

**Comment by itebj** [comment]
> @itebj Can you please take a look at the CI failure.

Yes, sir. I've removed some redundant definitions which caused errors.
Re-push after rebasing to the upstream/main as the current revision.

Thanks again for guidance.

@mmahadevan108 
@keith-zephyr 
@carlocaione 
@stephanosio

File: soc/riscv/riscv-ite/common/chip_chipregs.h [hunk 1]
```diff
@@ -10,17 +10,6 @@
...
-#ifndef FALSE
-#define FALSE   0
-#endif
-
-/* TRUE can be defined as !FALSE but defining
- * it as 1 allows it to fit into a bitfield.
- */
-#ifndef TRUE
-#define TRUE    1
-#endif
```
Comment (stephanosio) [inline soc/riscv/riscv-ite/common/chip_chipregs.h:22]: Please put this change in a separate commit.

Comment (itebj) [reply soc/riscv/riscv-ite/common/chip_chipregs.h:22]: @stephanosio 

Got it, sir.
I've put them in separate commits this time.
Thanks again.

**Review by stephanosio — APPROVED** [review]

**Comment by itebj** [comment]
For resolving the CI failure, the redundant definitions have been removed in this upload. Hence, there're two commits in this PR.

Thanks again for helps & guidance.

**Review by mmahadevan108 — APPROVED** [review]

**Pull request merged**